recipes/pixstem/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "pixstem" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 400da42fb11115b62548e8227f26b97c573870a72ebb5dbe66af437bec47c4a0
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python =>3
```

I guess this is supposed to be this?
```suggestion
    - python >=3
```